templates/tfengine/components/cicd/manual/configs/tf-plan.yaml

```diff
@@ -13,14 +13,24 @@
 # limitations under the License.
 
 timeout: 1200s
-options:
-    machineType: "N1_HIGHCPU_32"
 
 steps:
   - name: "gcr.io/cloud-foundation-cicd/cft/developer-tools@sha256:a09ad46f513847d31e2cfe20f818c2d64809e7d12695e2e6098900296f6527bf"
     entrypoint: terraform
     args: ["version"]
 
+  # org/cicd
+  - name: "gcr.io/cloud-foundation-cicd/cft/developer-tools@sha256:a09ad46f513847d31e2cfe20f818c2d64809e7d12695e2e6098900296f6527bf"
+    entrypoint: terraform
+    args: ["init"]
+    dir: "${_TERRAFORM_ROOT}/org/cicd"
```

Comment:
Shouldn't this be cicd/auto/? IIUC manual/ should be deployed manually, and auto/ will be deployed by CD.

Reply:
So it's the the directory in the generated Terraform configs, not the structure in the components. The generated configs will be structured as 

- bootstrap
- cicd
- org
  - cicd
  - ...